src/core/client/admin/routes/Configure/ConfigureLinks.tsx

```diff
@@ -18,6 +19,46 @@ const ConfigureLinks: FunctionComponent<{}> = () => {
           Moderation
         </Link>
       </Localized>
+      <Link
+        exact
+        to={{ pathname: "/admin/configure/moderation", hash: "#Comments" }}
+      >
+        {({
+          href,
+          onClick,
+        }: {
+          href: string;
+          onClick: (e: React.SyntheticEvent<Element>) => void;
```

Comment:
Doesn't really matter, but if you want, I believe there is a `MouseEvent` in React you can use that might be more descriptive/type accurate?

And if imported like so, will remove the `React.` part from the code.

```
import React, { FunctionComponent, MouseEvent } from "react";
```

Reply:
Improved this to `MouseEventHandler<HTMLButtonElement>`, which matches what `found` is expecting...thanks!